web/content/js/versions.js

```diff
@@ -123,6 +131,10 @@ const navVersions = [
             "title": "Installing Smart Proxy with Content",
             "path": "Installing_Proxy"
           },
+          {
+            "title": "Updating Katello",
```

Comment:
```suggestion
            "title": "Updating Katello to nightly",
```

Reply:
I wasn't sure to modify this until #2382 
Addressed :+1: